modules/core/client/config/core.client.routes.js

```diff
@@ -44,8 +44,7 @@
           }
         },
         data: {
-          ignoreState: true,
-          pageTitle: 'Not Found'
```

Comment:
What ends up being the title of specific user/article pages if those aren't found?